spec/integration/database_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+require 'spec/support/run_test_app'
+require 'active_record'
+
+describe 'Database setup', rails_min_version: 4 do
+  run_test_app
+  let(:app_options) {{ database: 'postgresql' }}
+
+  before { app.wait_start }
+
+  context 'When booting' do
+    before { app_helper.shell_run "cd #{app_path} && rake db:create" }
+    after  { app_helper.shell_run "cd #{app_path} && rake db:drop" }
```

Comment:
It didn't work, or rather risked leaking stuff between specs :)

The method was on `app_helper` which deals with the reusable scaffold tarball. It required keeping the clean scaffold uncompressed an ran the `rake` task there, instead of the 1-off uncompressed version used in each spec.